blog/2015/10/code-review-habit.html

```diff
@@ -0,0 +1,126 @@
+---
+layout: blog
+author: David Pérez-Suárez
+title: "Code Review - a needed habit in science"
+date: 2015-10-21
+time: "15:30:00"
+category: ["opinion"]
+---
+<!-- start excerpt -->
+<p>
+  More than a year ago, Marian Petre and Greg Wilson wrote an article
+about <a href="http://arxiv.org/abs/1407.5648">reviewing code</a>, but
+I was not aware of it till this week when Greg talked about their
+findings in a workshop in London
+(though <a href="{{page.root}}/blog/2014/01/code-review-round-2.html">it
+had been mentioned here before</a>).  I have been reviewing code for
+years, however I have not realised that I was doing so till GitHub
+made it easy. When I read code from others feels the same way that
```

Comment:
I'd change this to "Reading code from others feels the same as proofreading a Spanish text, ..."